charts/opensearch/CHANGELOG.md

```diff
@@ -13,6 +13,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 
+---
+## [1.0.3]
```

Comment:
Does this need to be 1.0.4?

Reply:
Thanks for catching this. Changes committed.